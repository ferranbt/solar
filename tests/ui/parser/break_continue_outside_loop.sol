contract C {
    function f() {
        break; //~ ERROR: `break` outside of a loop
        continue; //~ ERROR: `break` outside of a loop

        for (uint256 i = 0; i < 10; i++) {
            break;
            continue;
        }

        while (true) {
            break;
            continue;
        }

        do {
            break;
            continue;
        } while (true);
    }
}
