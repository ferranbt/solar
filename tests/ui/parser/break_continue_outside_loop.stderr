error: `break` outside of a loop
  --> ROOT/tests/ui/parser/break_continue_outside_loop.sol:LL:CC
   |
LL |         break;
   |         ^^^^^^
   |

error: `break` outside of a loop
  --> ROOT/tests/ui/parser/break_continue_outside_loop.sol:LL:CC
   |
LL |         continue;
   |         ^^^^^^^^^
   |

error: aborting due to 2 previous errors

